command/job_status.go

```diff
@@ -536,6 +538,60 @@ func (c *JobStatusCommand) outputJobSummary(client *api.Client, job *api.Job) er
 	return nil
 }
 
+// outputReschedulingEvals displays eval IDs and time for any
+// delayed evaluations by task group
+func (c *JobStatusCommand) outputReschedulingEvals(client *api.Client, job *api.Job, allocListStubs []*api.AllocationListStub, uuidLength int) error {
+	// Get the most recent alloc ID by task group
+
+	mostRecentAllocs := make(map[string]*api.AllocationListStub)
+	for _, alloc := range allocListStubs {
+		a, ok := mostRecentAllocs[alloc.TaskGroup]
+		if !ok || alloc.ModifyTime > a.ModifyTime {
+			mostRecentAllocs[alloc.TaskGroup] = alloc
+		}
+	}
+
+	followUpEvalIds := make(map[string]string)
+	for tg, alloc := range mostRecentAllocs {
+		if alloc.FollowupEvalID != "" {
+			followUpEvalIds[tg] = alloc.FollowupEvalID
+		}
+	}
+
+	if len(followUpEvalIds) == 0 {
+		return nil
+	}
+	// Print the reschedule info section
+
+	var delayedEvalInfos []string
+
+	taskGroups := make([]string, 0, len(followUpEvalIds))
+	for taskGroup := range followUpEvalIds {
+		taskGroups = append(taskGroups, taskGroup)
+	}
+	sort.Strings(taskGroups)
+	var evalDetails []string
+	for _, taskGroup := range taskGroups {
+		evalID := followUpEvalIds[taskGroup]
+		evaluation, _, err := client.Evaluations().Info(evalID, nil)
+		// Eval time is not critical output,
+		// so don't return it on errors, if its not set, or its already in the past
+		if err != nil || evaluation.WaitUntil.IsZero() || time.Now().After(evaluation.WaitUntil) {
+			continue
+		}
+		rp := job.LookupTaskGroup(taskGroup).ReschedulePolicy
+		evalTime := prettyTimeDiff(evaluation.WaitUntil, time.Now())
+		evalDetails = append(evalDetails, fmt.Sprintf("%s|%s|%s|%s", taskGroup, rp.String(), limit(evalID, uuidLength), evalTime))
+	}
+	if len(evalDetails) > 0 { // Only show this section if there is pending evals
+		delayedEvalInfos = append(delayedEvalInfos, "Task Group|Reschedule Policy|Eval ID|Eval Time")
```

Comment:
Lets limit showing the reschedule policy to verbose mode. I don't think we need it at all to be honest.